code/__HELPERS/game.dm

```diff
@@ -7,6 +7,8 @@
 
 #define Z_TURFS(ZLEVEL) block(locate(1,1,ZLEVEL), locate(world.maxx, world.maxy, ZLEVEL))
 
+#define HOLIDAY_ACTIVE(holiday)  SSevent.holidays && SSevent.holidays[##holiday]

```

Comment:
try wrapping all of the define in ()
eg:
```#define blah(x) (... your shit using x)```
The () are superfluous and won't break things, but may cause things to be evaluated correctly.